java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1127,6 +1127,67 @@ public ColumnVector dayOfYear() {
     return new ColumnVector(dayOfYear(getNativeView()));
   }
 
+  /**
+   * Rounding modes supported in round method.
+   * HALF_UP : Rounding mode to round towards "nearest neighbor". If both neighbors are
+   * equidistant, then round up.
+   * HALF_EVEN : Rounding mode to round towards the "nearest neighbor". If both neighbors are
+   * equidistant, round towards the even neighbor.
+   */
+  public enum RoundMode {
```

Comment:
I have put the Rounding mode enum in ColumnVector. I am still contemplating if this needs to be in a separate file as rounding method should be independent of ColumnVector.  Any suggestions are welcome.

Reply:
I would prefer to have it be a separate file if it is publicly exposed.  That is just because you cannot always predict what other ways it might be used in the future.

Reply:
Thanks @revans2 for your suggestion. Will update this.